apps/xplat-v9/README.md

```diff
@@ -38,6 +38,7 @@ Start the test app from the `apps/xplat-v9` directory:
 
 ```sh
 cd apps/xplat-v9
+pod install --project-directory=ios # or "macos" if you want to use them
```

Comment:
If this is only for ios/macos, it might be better to have it under a `#### iOS/macOS specific` heading.